src/app.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from "@nestjs/common";
```

Comment:
@PCalape nit: The quotes used between in this file and the other TS files are inconsistent, can we enforce once with Prettier or ESLint?

Reply:
Copy on this @arnellebalane. Will apply next commit